presto-docs/src/main/sphinx/release/release-0.282.rst

```diff
@@ -0,0 +1,47 @@
+=============
+Release 0.282
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix ``TEMPORARY`` defination :doc:`/sql/create-function` and :doc:`/sql/drop-function`.
+* Fix a bug where ``cardinality(map_keys(x))`` and ``cardinality(map_values(x))`` would return wrong results.
+* Fix several issues when building Presto on ARM based processors.
+* Fix schema support in resource group static selector.
+* Fix Explain (TYPE VALIDATE) to return after the analysis and ACL checks. It no longer executes the dummy query. It now returns the  ``true`` in the output column ``result`` instead of ``valid`` in case of successful execution.
+* Improve error handling when using custom ``FunctionNamespaceManagers``.
+* Improve null inferencing for join nodes. ``optimize_nulls_in_join`` session property is deprecated and is instead replaced with a new ``joins_not_null_inference_strategy`` session property (and corresponding configuration property ``optimizer.joins-not-null-inference-strategy``) to control null inferencing.
+* Add a new UDF :func:`array_cum_sum` to return the array whose elements are the cumulative sum of the input array.
+* Add a query optimization to rewrite left join with null check on right join key with semi join. It's controlled by session property `rewrite_left_join_null_filter_to_semi_join`.
+* Add an optimization for queries with empty input. The optimization is controlled by session property `simplify_plan_with_empty_input`.
```

Comment:
Improve queries with empty input.

Reply:
I would keep it as optimization only, unless author @feilong-liu wants to change it.